qa/rbd/krbd_discard_granularity.t

```diff
@@ -0,0 +1,40 @@
+
+  $ rbd create --size 4M img
+
+  $ DEV=$(sudo rbd map img)
+  $ blockdev --getiomin $DEV
+  65536
+  $ blockdev --getioopt $DEV
+  65536
+  $ cat /sys/block/${DEV#/dev/}/queue/discard_granularity
+  65536
+  $ sudo rbd unmap $DEV
+
+  $ DEV=$(sudo rbd map -o alloc_size=512 img)
+  $ blockdev --getiomin $DEV
+  512
+  $ blockdev --getioopt $DEV
+  512
+  $ cat /sys/block/${DEV#/dev/}/queue/discard_granularity
+  512
+  $ sudo rbd unmap $DEV
+
+  $ DEV=$(sudo rbd map -o alloc_size=4194304 img)
+  $ blockdev --getiomin $DEV
+  4194304
+  $ blockdev --getioopt $DEV
+  4194304
+  $ cat /sys/block/${DEV#/dev/}/queue/discard_granularity
+  4194304
+  $ sudo rbd unmap $DEV
+
+  $ DEV=$(sudo rbd map -o alloc_size=8388608 img)
```

Comment:
If I understand it correctly, this case is used to test alloc_size adjusted to object_size.  So maybe it's better to create image with object-size of 4M rather than image-size of 4M. e.g, "rbd create --size 50M --object-size 4M img".  :)

Reply:
Yes, it's the "adjust to object size" case.  I'll change the size of the image.